samples/notebooks/quickstart.ipynb

```diff
@@ -0,0 +1,572 @@
+    "\n",
+    "# Build and push the image.\n",
+    "cd tmp/components/list-gcs-blobs\n",
+    "bash build_image.sh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.2 Create a Python class for each component\n",
+    "Define a Python class that describes the interactions with the Docker container image created in the previous step. The Python class specifies the component name, the image to use, the command to run after the container starts, the input arguments, and the file outputs. Each component needs to inherit from `kfp.dsl.ContainerOp`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import kfp.dsl\n",
+    "\n",
+    "class ListGcsBlobsOp(kfp.dsl.ContainerOp):\n",
+    "  def __init__(self, name, bucket):\n",
+    "    super(ListGcsBlobsOp, self).__init__(\n",
+    "      name=name,\n",
+    "      image='gcr.io/{}/listgcsblobs:latest'.format(PROJECT_ID),\n",
+    "      command=['python', '/app/app.py'],\n",
+    "      file_outputs={'blobs': '/blobs.txt'},\n",
+    "      arguments=['--bucket', bucket]\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.3 Create your workflow as a Python function\n",
+    "Start by creating a folder to store the pipeline file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create folders if they don't exist.\n",
+    "!mkdir -p tmp/pipelines"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define your pipeline as a Python function. ` @kfp.dsl.pipeline` is a required decoration including `name` and `description` properties. Then compile the pipeline function. After the compilation is completed, a pipeline file is created."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime\n",
+    "import kfp.compiler as compiler\n",
+    "\n",
+    "# Define the pipeline\n",
+    "@kfp.dsl.pipeline(\n",
+    "  name='List GCS Blobs',\n",
+    "  description='Takes a GCS bucket name as input and lists the blobs.'\n",
+    ")\n",
+    "def pipeline_func(bucket=kfp.dsl.PipelineParam('bucket', value='Enter your bucket name here.')):\n",
+    "  list_blobs_task = ListGcsBlobsOp('List', bucket)\n",
+    "\n",
+    "# Compile the pipeline to a file.\n",
+    "filename = 'tmp/pipelines/list_blobs{dt:%Y%m%d_%H%M%S}.pipeline.tar.gz'.format(dt=datetime.datetime.now())\n",
+    "compiler.Compiler().compile(pipeline_func, filename)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Go to your Cloud Shell, run `conn.sh`, click on the provided link. In the new tab that opens, click on \"Pipelines Dashboard\" and go to Kubeflow pipelines UI. Upload the created pipeline and run it.\n",
```

Comment:
Can we make these instructions more portable? That is, rather than use cloud shell, let's just ask user to do port forwarding. You can link here to the instructions on the main kubeflow.org site.

Reply:
Done!